pkg/apis/kops/componentconfig.go

```diff
@@ -413,6 +413,9 @@ type KubeAPIServerConfig struct {
 	// The specified file can contain multiple keys, and the flag can be specified multiple times with different files.
 	// If unspecified, --tls-private-key-file is used.
 	ServiceAccountKeyFile []string `json:"serviceAccountKeyFile,omitempty" flag:"service-account-key-file"`
+
+	// CPURequest, cpu request compute resource for kube proxy e.g. "20m"
```

Comment:
Maybe we should include the default here instead of the e.g.? `Defaults to "150m"`

Reply:
Comment updated with the default and update to say api server instead of kube proxy.